spec/spec_helper_acceptance_local.rb

```diff
@@ -44,15 +44,15 @@ def update_profile_file
 end
 
 RSpec.configure do |c|
+  # This flag is disabling some tests on docker/vagrant containers
+  # To enable tests on abs/vmpooler machines just set to `true` this flag
+  c.filter_run_excluding ubuntu_vmpooler: false
```

Comment:
I'm not entirely sure I understand this - the comment states that the flag disables tests on Docker/Vagrant, but is then controlling vmpooler tests?

Reply:
exactly